models/comments.model.js

```diff
@@ -21,4 +22,19 @@ exports.getCommentsForArticle = ( articleId ) => {
             return comments
         }
     })
+}
+
+exports.addCommentToArticleForUser = (comment, article_id, username) => {
+    if (!username || !comment) return Promise.reject({ status: 400, msg: 'Missing inputs' })
+    const formattedQuery = format(`
+        INSERT INTO comments
+            (body, article_id, author)
+        VALUES
+            %L
+        RETURNING comment_id, votes, created_at, author, body, article_id
```

Comment:
could you not just use RETURNING * here??

Reply:
My thinking was this could expose data in future, if some columns should not be shared with the client. Would you suggest in general to use * and then add the behaviour of restricted columns if that comes up?